secant/Screens/Onboarding/OnboardingContentView.swift

```diff
@@ -61,34 +61,76 @@ struct OnboardingContentView: View {
                     VStack(spacing: viewStore.isFinalStep ? 50 : 15) {
                         HStack {
                             Text(viewStore.steps[stepIndex].title)
-                                .font(.custom(FontFamily.Roboto.bold.name, size: 30))
-                                .fontWeight(.regular)
+                                .titleText()
+                                .lineLimit(0)
+                                .minimumScaleFactor(0.1)
                             if !viewStore.isFinalStep {
                                 Spacer()
                             }
                         }
                         
                         Text(viewStore.steps[stepIndex].description)
-                            .font(.custom(FontFamily.Roboto.regular.name, size: 15))
-                            .lineSpacing(5)
+                            .synopsisText()
+                            .lineSpacing(2)
+                            .opacity(0.53)
                     }
                     .opacity(stepIndex == viewStore.index ? 1: 0)
                     .padding(.horizontal, 35)
                     .frame(width: width, height: height)
                 }
             }
-            .offset(y: viewStore.isFinalStep ? width / 2.3 : viewStore.offset + width / 2.3)
+            .offset(y: viewStore.isFinalStep ? width / 2.5 : viewStore.offset + height / descriptionOffsetCoeff())
         }
     }
 }
 
+/// Following computations are necessary to handle properly sizing and positioning of elements
+/// on different devices (apects). iPhone SE and iPhone 8 are similar aspect family devices
+/// while iPhone X, 11, etc are different family devices, capable to use more of the space.
+extension OnboardingContentView {
```

Comment:
I think these can all be computed properties, because they don't take input and just project a value.  Because the calcite is also using constant data it might read better that way.  It may even be possible that making the computed property an assignment has performance benefits if the view is recalculated with new content?

Reply:
updated